scripts/purge_queue.py

```diff
@@ -0,0 +1,128 @@
+            queue_attributes = queue['Attributes']
+            print('{:<35s}{:^25s}{:^20s}{:^20s}'.format(queue_name,
+                                                        queue_attributes['ApproximateNumberOfMessages'],
+                                                        queue_attributes["ApproximateNumberOfMessagesNotVisible"],
+                                                        queue_attributes["ApproximateNumberOfMessagesDelayed"]))
+
+
+def get_messages_from_queue(options):
+    sqs = boto3.resource("sqs")
+    logger = logging.getLogger(__name__)
+    queue_name = sqs.get_queue_by_name(QueueName=options.queue_name).url
+    output_file_name = options.file_name
+    queue = sqs.Queue(url=queue_name)
+    logger.info("Obtaining messages from queue '%s'", queue_name)
+    logger.info("Writing messages to '%s'", output_file_name)
+
+    visibility_timeout = 60 * 5
+    logger.info("Default visibility timeout '%i'", visibility_timeout)
+    messages = []
+    while True:
+        messages_batch = queue.receive_messages(AttributeNames=['All'],
+                                                MaxNumberOfMessages=10,
+                                                VisibilityTimeout=visibility_timeout)
+        if not messages_batch:  # Nothing left in queue
+            break
+        else:
+            messages.extend(messages_batch)
+    with open(output_file_name, "w") as file:
+        json.dump({"queue": queue_name,
+                   "messages": extract_fields(messages, options.json_body)},
+                  fp=file,
+                  indent=4)
+    logger.info("Wrote %i messages", len(messages))
+    message_batches = list(more_itertools.chunked(messages, 10))
+    if options.delete:
+        logger.info("Removing messages from queue '%s'", queue_name)
+        delete_messages(message_batches, queue)
+    else:
+        logger.info("Return messages to `%s` queue", queue_name)
+        return_messages(message_batches, queue)
+    logging.info("File: {}\nWritten: OK!".format(output_file_name))
+
+
+def return_messages(message_batches, queue):
+    for message_batch in message_batches:
+        batch = [dict(Id=message.message_id,
+                      ReceiptHandle=message.receipt_handle,
+                      VisibilityTimeout=0) for message in message_batch]
+        response = queue.change_message_visibility_batch(Entries=batch)
+        if len(response['Successful']) != len(batch):
+            raise RuntimeError(f"Failed to revert visibility: resp={response!r}")
+
+
+def delete_messages(message_batches, queue_resource):
+    for message_batch in message_batches:
+        response = queue_resource.delete_messages(
+            Entries=[dict(Id=message.message_id,
+                          ReceiptHandle=message.receipt_handle) for message in message_batch])
+        if len(response['Successful']) != len(message_batch):
+            raise RuntimeError(f"Failed to delete messages: resp={response!r}")
```

Comment:
```suggestion
            raise RuntimeError(f"Failed to delete messages: {response!r}")
```

Here and everywhere else.